proposal/Luke2336/README.rst

```diff
@@ -0,0 +1,59 @@
+===================
+Graph Visualization
+===================
+
+
+Basic Information
+=================
+
+- GitHub repository: https://github.com/Luke2336/graph-visualization
+
+Problem to Solve
+================
+
+Visualizing a graph can help people study graph theory. In this project, I would like to use force-directed method (Fruchterman-Reingold Algorithm) to layout a simple undirected graph to let the number of edge intersections as small as possible.
+
+
+Prospective Users
+=================
+
+The prospective users are poeple who want to visualize a graph.
+
+System Architecture
+===================
+
+- Input format
+	The first line contains two non-negative integers n and m.
+	For 2 ~ m + 1 lines, there are two integer a and b, which means there is an edge between vertex a and b.
+- Flow chart
```

Comment:
Input format and flow chart have less to do with the computation you need to carry out.  Important information is missed:

* What will be the data structure used for the graph?

* How large will the graph be?